website/docs/cloud-docs/api-docs/run.mdx

```diff
@@ -90,11 +91,12 @@ The run operation specifies the Terraform execution mode. You can reference the
 
 | Operation        | Description                                                                                                                                                                |
 |------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `plan_only`      | Indicates the run completed without an apply phase. This is also called a [speculative plan](/terraform/cloud-docs/run/modes-and-options#plan-only-speculative-plan).                |
-| `plan_and_apply` | Indicates the run completed with both plan and apply phases.                                                                                                               |
-| `refresh_only`   | Specifies the run was instructed to execute a plan to update Terraform state.                                                                                                |
-| `destroy`        | Specifies the run destroyed all objects, regardless of configuration changes.                                                                                              |
-| `empty_apply`    | Specifies the run performed an apply with no changes to resources. This is most commonly used to [upgrade terraform state versions](/terraform/cloud-docs/workspaces/state#upgrading-state). |
+| `plan_only`      | The run does not have an apply phase. This is also called a [speculative plan](/terraform/cloud-docs/run/modes-and-options#plan-only-speculative-plan).                |
+| `plan_and_apply` | The run includes both plan and apply phases.                                                                                                               |
+| `save_plan`      | The run is a saved plan run; it can include both plan and apply phases, but won't become the workspace's current run unless it is confirmed for apply.                     |
```

Comment:
Small suggestion for active voice and to avoid using a semicolon outside of code:
```suggestion
| `save_plan`      | The run is a saved plan run and can include both plan and apply phases, but it will only become the workspace's current run if you confirm you want to apply it.                     |
```

Reply:
Aww, but I like my semis 😂 —hmm, I'm not a fan of "and...but" here, so let's just split it to sentences.